bootique/src/test/java/io/bootique/Bootique_CliOptionsIT.java

```diff
@@ -445,6 +477,36 @@ public CommandOutcome run(Cli cli) {
         }
     }
 
+    static final class CommandAWithDefaultValue extends CommandWithMetadata {
+        public CommandAWithDefaultValue() {
+            super(CommandMetadata.builder("commandDefaultValue")
+                    .valueOptionalWithDefault("commandVal")
+                    .shortName('A')
```

Comment:
I had some trouble getting the CLI to recognize command names without the use of the `shortName`. Not strictly necessary, but helped get the tests green. Would like to test without the `shortName` if you know how to get there.